example.py

```diff
@@ -41,9 +41,9 @@
 
 print(posthog.feature_enabled("beta-feature", "distinct_id"))
 
-# # Alias a previous distinct id with a new one
+# Add an alias_id to a distinct_id
```

Comment:
This is not clear to me what alias_id restrictions are.

Reply:
I thought we are removing the alias restrictions with the new updates of persons on events enabling person merging?